ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -60,6 +60,12 @@ EOF
     )
 fi
 
+echo "Setting label security.openshift.io/scc.podSecurityLabelSync value to true on the namespace \"$OO_INSTALL_NAMESPACE\""
```

Comment:
I'd move this echo to be within the `if` block for clarity

Reply:
fixed here: https://github.com/openshift/release/pull/31900